src/3d/qgscameracontroller.h

```diff
@@ -229,8 +229,17 @@ class _3D_EXPORT QgsCameraController : public QObject
     {
       None = 0,
       Translation,
-      Rotation,
-      Zoom
+      RotationCamera,
+      RotationCenter,
+      Zoom,
+      ZoomWheel
+    };
+
+    const QList<MouseOperation> mTranslateOrRotate =
+    {
+      MouseOperation::Translation,
+      MouseOperation::RotationCamera,
+      MouseOperation::RotationCenter
```

Comment:
Can you add a comment to explain the use of this list?

Reply:
Done.